modules/core/src/lib/attribute-transition-manager.js

```diff
@@ -120,7 +120,7 @@ export default class AttributeTransitionManager {
         name: attributeName,
         timeline: this.timeline,
         attribute,
-        attributeInTransition: new Attribute(this.gl, attribute),
+        attributeInTransition: new Attribute(this.gl, attribute.userData),
```

Comment:
Why are we making this change?

Reply:
`attribute.userData` is the original options passed when constructing `attribute`. This ensures that we set the proper `doublePrecision` flag and shader attributes.
Added a comment.

Reply:
Creating an attribute with `attribute.userData` as the `opts` doesn't appear to transfer the attribute's `buffer` or the `value`. Is this intentional?

Reply:
Yes, `attributeInTransition` is supposed to use the feedback buffer.

Reply:
That comment you added is super helpful! Thank you!